core/templates/pages/library-page/library-footer/library-footer.component.ts

```diff
@@ -21,20 +21,23 @@ import { downgradeComponent } from '@angular/upgrade/static';
 import { WindowRef } from 'services/contextual/window-ref.service';
 import { LibraryPageConstants } from '../library-page.constants';
 
+type libraryPathToModesKeys = (
+  keyof typeof LibraryPageConstants.LIBRARY_PATHS_TO_MODES);
```

Comment:
Again capitalize the first letter of the type.

Reply:
Done